src/main/resources/gtfs/gtfs.yml

```diff
@@ -581,6 +581,16 @@
     inputType: GTFS_STOP
     columnWidth: 6
     helpContent: "The stop_id field contains an ID that uniquely identifies a stop. Multiple routes may use the same stop. The stop_id is referenced from the stops.txt file. If location_type is used in stops.txt, all stops referenced in stop_times.txt must have location_type of 0."
+  - name: "location_id"
+    required: false
+    inputType: GTFS_ID
+    columnWidth: 6
+    helpContent: "The location_id field contains an ID that uniquely identifies a location. Multiple routes may use the same stop. The stop_id is referenced from the stops.txt file. If location_type is used in stops.txt, all stops referenced in stop_times.txt must have location_type of 0."
```

Comment:
"Multiple routes may use the same *location*". And also, please clarify "The stop_id is referenced from the stops.txt file." (This field is about `location_id`)

Reply:
@binh-dam-ibigroup On review I should probably revert this back and create a new gtfsFlex.yml file. However, this current file is only referenced once in DT and the new gtfsFlex.yml wouldn't be used. So, I'm just going to revert this file back and not create the new file. Thoughts?

Reply:
Sure

Reply:
Fixed in: https://github.com/ibi-group/datatools-server/pull/599/commits/10d924c72b5e18346ba2146ddb90601d35eda486